src/canvas-force-graph.js

```diff
@@ -116,17 +118,22 @@ export default Kapsule({
       function paintNodes() {
         const getVal = accessorFn(state.nodeVal);
         const getColor = accessorFn(state.nodeColor);
+        const getNodeCanvasObjectMode = state.nodeCanvasObject ? accessorFn(state.nodeCanvasObjectMode) : () => undefined;
```

Comment:
This can actually be simplified to just `accessorFn(state.nodeCanvasObjectMode)`, because calling `accessorFn(undefined)` returns `() => undefined`. 😄